lib/client-api/src/story_store.ts

```diff
@@ -481,7 +484,14 @@ export default class StoryStore {
 
   setError = (err: ErrorLike) => {
     this._error = err;
-    if (this._channel) this._channel.emit(Events.RENDER_CURRENT_STORY);
+
+    if (this._channel) {
+      // If the selection is set while configuration is in process, we are guaranteed
+      // we'll emit RENDER_CURRENT_STORY at the end of the process, so we shouldn't do it now.
+      if (!this._configuring) {
+        this._channel.emit(Events.RENDER_CURRENT_STORY);
+      }
+    }
```

Comment:
I don't think it is allowed that you set error outside of configuration. In fact we should probably throw if you do that.

Reply:
AFAIK the setError is also called when a story throws?

Reply:
Who by?